app/controllers/candidate_interface/start_page_controller.rb

```diff
@@ -15,9 +15,9 @@ def create_account_or_sign_in_handler
       render :create_account_or_sign_in and return unless @create_account_or_sign_in_form.valid?
 
       if @create_account_or_sign_in_form.existing_account?
-        redirect_to candidate_interface_sign_in_path(
-          providerCode: params[:providerCode],
-          courseCode: params[:courseCode],
+        SignInCandidate.call(
```

Comment:
I would skip the `call` and call `new` here, I think it saves some code in the service object and makes it easier to refactor (you only have to rename a param in 2 places).